Kudu.Services.Web/Startup.cs

```diff
@@ -480,6 +485,34 @@ private static Uri GetAbsoluteUri(HttpContext httpContext)
                     new {controller = "Deployment", action = "IsDeploying"},
                     new {verb = new HttpMethodRouteConstraint("GET")});
 
+                // Initiate Scan 
+                routes.MapRoute("start-clamscan-timeout", "api/scan/start/{timeout}",
```

Comment:
timeout should ideally be a Query param or a POST param or a query param. Since the number in start/number doesn't describe in the URL that it is a timeout value.

Reply:
Made it a query param.